src/wallet/wallet.cpp

```diff
@@ -4045,6 +4376,77 @@ string CWallet::SpendZerocoin(std::string &thirdPartyaddress, int64_t nValue, li
     return "";
 }
 
+
+/**
+ * @brief CWallet::SpendZerocoin
+ * @param thirdPartyaddress
+ * @param nValue
+ * @param denomination
+ * @param wtxNew
+ * @param coinSerial
+ * @param txHash
+ * @param zcSelectedValue
+ * @param zcSelectedIsUsed
+ * @return
+ */
+string CWallet::SpendMultipleZerocoin(std::string &thirdPartyaddress, std::vector<std::pair<int64_t, libzerocoin::CoinDenomination>> denominations, CWalletTx &wtxNew,
+                              CBigNum &coinSerial, uint256 &txHash, CBigNum &zcSelectedValue,
+                              bool &zcSelectedIsUsed, bool forceUsed) {
+    // Check amount
+    // TODO do for all
+    // if (nValue <= 0)
+    //     return _("Invalid amount");
+     // Do not allow spend to take place until fully synced
+    // Temporary measure: we can remove this limitation when well after spend v1.5 HF block
+    // if (fImporting || fReindex || !znodeSync.IsBlockchainSynced())
+    //     return _("Not fully synced yet");
+     CReserveKey reservekey(this);
+     if (IsLocked()) {
+        string strError = _("Error: Wallet locked, unable to create transaction!");
+        LogPrintf("SpendZerocoin() : %s", strError);
```

Comment:
I think it need to be `strError.c_str()` instead. I don't known what type that is returned from `_()` but if it is `char *` then we can switch `strError` to `const char *` instead to increase performance.